less/index.less

```diff
@@ -57,3 +58,271 @@ html, body {
     height: 100%;
 }
 
+//Theme Variables
+@secondary-color: @apricot; 
```

Comment:
Great job using themes. Is this what you learned with your UI/UX background? Please share the knowledge with the group, we'll  all be grateful!